src/LinuxTracing/PerfEvent.h

```diff
@@ -259,6 +259,118 @@ struct DmaFenceSignaledPerfEventData {
 };
 using DmaFenceSignaledPerfEvent = TypedPerfEvent<DmaFenceSignaledPerfEventData>;
 
+struct CallchainSchedWakeupPerfEventData {
```

Comment:
I suggest to follow the Uprobes naming schemes and go with `SchedWakeupWithCallchainPerfEventData` (same below)

Reply:
Done.